hknweb/candidate/templates/candidate/bitbyte_confirm_email.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>title</title>
```

Comment:
I'm not actually sure what the title tag is used for in a HTML email, even after looking online a bit. Anyway, I'd say either add a more useful title here or remove the element entirely (Same with the other email template)

Reply:
I found this https://litmus.com/community/discussions/1521-how-useful-is-having-a-title-tag-in-email-html

Reply:
I added the titles